evm-tests/jsontests/Cargo.toml

```diff
@@ -11,19 +11,19 @@ edition.workspace = true
 [dependencies]
 evm.workspace = true
 ethereum = "0.15.0"
-primitive-types = "0.12"
+primitive-types = "0.13"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 hex = "0.4"
 clap = { version = "4.5", features = ["cargo"] }
 ethjson = { path = "../ethjson", features = ["test-helpers"] }
 libsecp256k1 = "0.7"
 ethcore-builtin = { path = "../ethcore-builtin" }
-rlp = "0.5"
+rlp = "0.6"
 sha3 = "0.10"
 parity-bytes = "0.1"
 env_logger = "0.11"
-lazy_static = "1.4.0"
+lazy_static = "1.5"
```

Comment:
Maybe we can get rid of this dependency by using the `LazyCell`?

Reply:
It will be part of refactoring for `jsontests`.